addons/power/functions/fnc_solarCalculation.sqf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Calculates the current solar panel output.
+ *
+ * Arguments:
+ * 0: Solarpanel <OBJECT>
+ *
+ * Returns:
+ * 0: Powerstate <BOOL>
+ * 1: Poweroutput <FLOAT>
+ */
+
+params['_solar'];
+
+call AE3_power_fnc_getSolarPosition params ['_solZen', '_solAz'];
+private _solVec = [cos _solZen * cos _solAz, cos _solZen * sin _solAz, sin _solZen];
+
+private _normalList = [_solar] call (_solar getVariable "AE3_power_orientationFnc");
+private _mult = 0;
+
+{
+	_buffer = _x vectorCos _solVec;
+	if (_buffer > 0) then 
+	{
+		_mult = _mult + _buffer;
+	};
+}forEach _normalList;
+
+// If sun is visible
+private _solPos = getPosASL _solar vectorAdd (_solVec vectorMultiply 10);
```

Comment:
//Move down one line and modify likes this:
private _solPos = _panelPos vectorAdd (_solVec vectorMultiply 10);

Reply:
vectorMultiplay 100 would be better